### ISSUE TITLE
Source code of tiny lib

### ISSUE THREAD

**Issue by jaredsburrows** [Sunzxyong/Tiny#12]
Where is the c/c++ JNI source code of the tiny lib?

**Comment by Sunzxyong** [comment]
You can see the [sample code](https://github.com/Sunzxyong/Tiny/blob/master/sample/example.c),also see [my blog](http://zhengxiaoyong.me/2017/04/23/%E4%B9%9F%E8%B0%88%E5%9B%BE%E7%89%87%E5%8E%8B%E7%BC%A9/).

**Comment by jaredsburrows** [comment]
Yeah. Just without the whole repo being open source and transparent, people may or may not want to use it. Also what is the license of the tiny C, C++ lib?
